telcell/celldb/__main__.py

```diff
@@ -0,0 +1,84 @@
+import sys
+from functools import partial
+from typing import Sequence
+
+import click
+from tqdm import tqdm
+
+from telcell.celldb import pgdatabase
+from telcell.utils import script_helper
+
+
+def create_antenna_light_table(con):
+    pgdatabase.create_table(con)
+    tablename = "antenna_light"
+    with con.cursor() as cur:
+        cur.execute(
+            f"""
+            INSERT INTO {tablename}(date_start, date_end, radio, mcc, mnc, lac, ci, eci, rd, azimuth)
```

Comment:
eerder hadden we hier ook antenna_id, zipcode, city nog bij (om later in de measurements te stoppen zodat je ze goed weg kan schrijven in de write_lrs functie

Reply:
Kan inderdaad handig zijn, maar nu niet makkelijk erbij te krijgen